zcash_client_backend/src/fees.rs

```diff
@@ -76,20 +93,25 @@ impl ChangeValue {
         Self::shielded(ShieldedProtocol::Orchard, value, memo)
     }
 
-    /// Returns the pool to which the change output should be sent.
+    /// Returns the pool to which the change or ephemeral output should be sent.
     pub fn output_pool(&self) -> PoolType {
         self.output_pool
     }
 
-    /// Returns the value of the change output to be created, in zatoshis.
+    /// Returns the value of the change or ephemeral output to be created, in zatoshis.
     pub fn value(&self) -> NonNegativeAmount {
         self.value
     }
 
-    /// Returns the memo to be associated with the change output.
+    /// Returns the memo to be associated with the output.
     pub fn memo(&self) -> Option<&MemoBytes> {
         self.memo.as_ref()
     }
+
+    /// Whether this is to be an ephemeral output.
+    pub fn is_ephemeral(&self) -> bool {
```

Comment:
Same question here: should this method be behind the `transparent-inputs` feature?

Reply:
I don't see that it needs to be.